test/PowerShellEditorServices.Test.Host/TestFiles/CompleteFunctionName.ps1

```diff
@@ -4,4 +4,6 @@ function My-Function
 $Cons
 My-
 Get-Proc
-$HKC
\ No newline at end of file
+$HKC
+Get-ChildItem ./TestFiles/Folder
+Get-ChildItem ./TestFiles/CompleteF
```

Comment:
Do you want to add a test to make sure that it continues to complete after it resolves? In otherwords:

```pwsh
Get-ChildItem ./TestFiles/Folder
```
goes to:
```pwsh
Get-ChildItem './TestFiles/Folder With Spaces'
```
but in theory it should keep going:
```pwsh
Get-ChildItem './TestFiles/Folder With Spaces/AnotherFolder'
```

Reply:
It won't go to that unless you type a `/` char.  The main point of this point is to put the cursor at the right spot so you only have to type a `/` to get further auto-completion.  BTW this behaves differently than the command-line completion where it automatically adds the `/` char.  I "think" we don't get that in PSES because we use the AST for completions but not 100% sure on that.

Reply:
interesting! Thanks for sharing. Might be still worth having a test for this:

```
Get-ChildItem './TestFiles/Folder With Spaces/AnotherFolder/'
```

and that completions should still show up after the last `/`?

Reply:
The way the tests work, you setup a partial completion in a file, then send the completion request for that spot in the file.  I could set up another line in the file to check but I don't how that would add value over the two lines I've added for testing folder and file completion.

Reply:
Yeah what we have is fine.